integration-tests/admission/admission_suite_test.go

```diff
@@ -1,138 +1,13 @@
-package admission
+package admission_test
```

Comment:
Ref: [Package names in golang](https://blog.golang.org/package-names)
This should give linting errors.
Tl; dr from reference.
```
The style of names typical of another language might not be idiomatic in a Go program. Here are two examples of names that might be good style in other languages but do not fit well in Go:

    computeServiceClient
    priority_queue
```
@golangcibot should have mentioned this